lib/pointing_party_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule PointingPartyWeb.SessionController do
+  use PointingPartyWeb, :controller
+  alias PointingParty.Account.Auth
+  alias PointingParty.Account
+
+  def new(conn, _params) do
+    changeset = Account.changeset(%Account{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, params) do
+    case Auth.login(params["account"]) do
+      {:ok, account} ->
```

Comment:
What do you think of using pattern matching to pull out the username and cleaning up the pipeline a tad?

```elixir
def create(conn, params) do
  case Auth.login(params["account"]) do
    {:ok, %{username: username}} ->
      conn
      |> put_session(:username, username)
      |> redirect(to: "/cards") 
      |> halt()

    {:error, changeset} ->
      render(conn, "new.html", changeset: changeset)
  end
end
```